test/container/php/test_config.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+CONFIG_DIR = '/usr/local/etc/php'
+
+@pytest.mark.php
+def test_php_ini_dir_environment_variable_is_set(host):
+    assert CONFIG_DIR in host.run('echo $PHP_INI_DIR').stdout
+
+@pytest.mark.php
+def test_php_config_files_are_present(host):
+    assert host.file(CONFIG_DIR + '/php.ini-development').exists is True
+    assert host.file(CONFIG_DIR + '/php.ini-production').exists is True
+
+@pytest.mark.php
+def test_php_base_config_is_present(host):
+    assert host.file(CONFIG_DIR + '/php.ini').is_symlink is True
+
+@pytest.mark.php_no_dev
+def test_production_config_is_linked_to(host):
+    host.file(CONFIG_DIR + '/php.ini').linked_to is CONFIG_DIR + '/php.ini-production'
+
+@pytest.mark.php_dev
+def test_development_config_is_linked_to(host):
+    host.file(CONFIG_DIR + '/php.ini').linked_to is CONFIG_DIR + '/php.ini-development'
+
+@pytest.mark.php
+def test_config_files_are_loaded(host):
+    files = get_loaded_config_files(host)
+
+    assert '/usr/local/etc/php/php.ini' in host.run('php --ini').stdout
```

Comment:
Isn't this host run already being done in the line above?

Reply:
Woops, missed that line.